mod-circulation/src/main/resources/vega/mod-circulation/features/loans.feature

```diff
@@ -937,4 +937,59 @@ Feature: Loans tests
     Given path '/_/proxy/tenants/' + tenant + '/timers'
     And request '{"id":"mod-circulation_6","routingEntry":{"unit": "minute", "delay":"30"} }'
     When method PATCH
-    Then status 204
\ No newline at end of file
+    Then status 204
+
+  Scenario: When an overdue item is renewed, an overdue fine is billed per the Overdue Fine Policy
+    * def extItemBarcode = 'FAT-1018IBC'
```

Comment:
Please, add empty line above current and under scenario name

Reply:
fixed, thanks